<?xml version="1.0" encoding="utf-8"?>
<launch>
  <node pkg="state_machine" type="test_bvp_node" name="test_bvp_node" output="screen">
    <param name="test/start_px" value="0.0" type="double" />
    <param name="test/start_py" value="0.0" type="double" />
    <param name="test/start_pz" value="0.0" type="double" />
    <param name="test/start_vx" value="-2.0" type="double" />
    <param name="test/start_vy" value="-2.0" type="double" />
    <param name="test/start_vz" value="0.0" type="double" />
    <param name="test/start_ax" value="0.0" type="double" />
    <param name="test/start_ay" value="0.0" type="double" />
    <param name="test/start_az" value="0.0" type="double" />
    <param name="test/rho" value="0.01" type="double" />
  </node>
  
</launch>
